cmd/gcp-controller-manager/main.go

```diff
@@ -87,7 +87,8 @@ func main() {
 		LeaseDuration: metav1.Duration{Duration: 15 * time.Second},
 		RenewDeadline: metav1.Duration{Duration: 10 * time.Second},
 		RetryPeriod:   metav1.Duration{Duration: 2 * time.Second},
-		ResourceLock:  rl.EndpointsResourceLock,
+		//TODO(acumino): Switch endpointsleases to leases in one of the future version.
```

Comment:
be a little more prescriptive about when this can happen safely (e.g. in which version or in what timeframe)